HonestHouse/HonestHouse/Sources/Core/Model/4.3. Camera Information/1. CameraFixedInformation.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  1. CameraFixedInformation.swift
```

Comment:
크게 영향을 미치진 않지만 이 부분 다른 파일들은 숫자 없이 되어 있어서 통일되게 가면 좋을 거 같슴다!!
```suggestion
//  CameraFixedInformation.swift
```